src/libYARP_manager/src/localbroker.cpp

```diff
@@ -978,7 +977,10 @@ int LocalBroker::ExecuteCmd()
         close(pipe_child_to_parent[WRITE_TO_PIPE]);
         FILE* in_from_child = fdopen(pipe_child_to_parent[READ_FROM_PIPE],"r");
         int flags=fcntl(pipe_child_to_parent[READ_FROM_PIPE],F_GETFL,0);
-        fcntl(pipe_child_to_parent[READ_FROM_PIPE],F_SETFL,flags|O_NONBLOCK);
+        if (fcntl(pipe_child_to_parent[READ_FROM_PIPE],F_SETFL,flags|O_NONBLOCK) != 0)
+        {
+            return -1;
```

Comment:
Why `-1` as a return value? It does not seems to be used in the rest of the function anywhere, and is not documented in the function signature.

Reply:
Right I didn't check carefully. I see there are similar situations in the same function at https://github.com/robotology/yarp/blob/4bca59c4f41bff16e9b4d453257878ab263745be/src/libYARP_manager/src/localbroker.cpp#L879
and 
https://github.com/robotology/yarp/blob/4bca59c4f41bff16e9b4d453257878ab263745be/src/libYARP_manager/src/localbroker.cpp#L895
so I'll do the same here.